licenses.d2iq.yaml

```diff
@@ -1,27 +1,12 @@
 ignore:
   - docker.io/mesosphere/grafana-plugins:v0.0.1
-
-  # The image is set of tools that is not built from source code.
```

Comment:
i remember you were talking about this - have we determined that having comments is not possible in the yaml generator ?

Reply:
@takirala hypothetically it could be possible but it was decided it'd be more effort than benefit given so decided to just add to README